### PR TITLE
Added docs for SCT feature Unity XT

### PR DESCRIPTION
**Pull request by rajendraindukuri**
# Description
Documentation related to SCT changes

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/876|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/features/unity.md [hunk 1]
```diff
@@ -492,6 +492,19 @@ This feature:
...
+If such a node is not available, the pods stay in Pending state. This means pods are not scheduled.
+
+Without storage capacity tracking, pods get scheduled on a node satisfying the topology constraints. If the required capacity is not available, volume attachment to the pods fails, and pods remain in ContainerCreating state. Storage capacity tracking eliminates unnecessary scheduling of pods when there is insufficient capacity.
+
+The attribute `storageCapacity.enabled` in `values.yaml` can be used to enable/disable the feature during driver installation using helm. This is by default set to true. To configure how often driver checks for changed capacity set `storageCapacity.pollInterval` attribute. In case of driver installed via operator, this interval can be configured in the sample file provided [here.](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_unity_v280.yaml) by editing the `--capacity-poll-interval` argument present in the provisioner sidecar.
+
+**Note:**
+>This feature requires kubernetes v1.24 and above and will be automatically disabled in lower version of kubernetes.
```
Comment (rensyct) [inline content/docs/csidriver/features/unity.md:506]: Do we need to mention this part of the above statement _"and will be automatically disabled in lower version of kubernetes_." and versions below 1.26 of K8s is not in support matrix

File: content/docs/csidriver/installation/operator/unity.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ Users should configure the parameters in CR. The following table lists the prima
    | X_CSI_HEALTH_MONITOR_ENABLED                    | Enable/Disable health monitor of CSI volumes from Node plugin               | No       |                       |
 
 ### Example CR for Unity XT
-Refer samples from [here](https://github.com/dell/dell-csi-operator/tree/master/samples). Below is an example CR:
+Refer samples from [here](https://github.com/dell/csm-operator/blob/main/samples). Below is an example CR:
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/unity.md:101]: This section is specific to dell-csi-operator. So please revert the change in this file
For driver installed via csm-operator, please add the required changes in https://dell.github.io/csm-docs/docs/deployment/csmoperator/drivers/unity/

**Review by shanmydell — DISMISSED** [review]

File: content/docs/csidriver/release/unity.md [hunk 3]
```diff
@@ -3,15 +3,11 @@ title: Unity XT
...
+## Release Notes - CSI Unity XT v2.8.0
 
 ### New Features/Changes
 
-- [Migrated image registry from k8s.gcr.io to registry.k8s.io](https://github.com/dell/csm/issues/744)
-- [Added support for OpenShift 4.12](https://github.com/dell/csm/issues/571)
-- [Added support for Kubernetes 1.27](https://github.com/dell/csm/issues/761)
-- [Added support for K3s on Debian OS](https://github.com/dell/csm/issues/798)
-- [Added support for Unisphere 5.3.0 array](https://github.com/dell/csm/issues/842)
+- [Added support for Storage Capacity Tracking for Unity XT](https://github.com/dell/csm/issues/876)
```
Comment (rensyct) [inline content/docs/csidriver/release/unity.md:11]: Please add this under release notes for csm-operator as well

File: content/docs/csidriver/features/unity.md (hunk 1 above)
Comment (rajendraindukuri) [reply content/docs/csidriver/features/unity.md]: @rensyct removed this note completely

File: content/docs/csidriver/installation/operator/unity.md (hunk 2 above)
Comment (rajendraindukuri) [reply content/docs/csidriver/installation/operator/unity.md]: - done

File: content/docs/csidriver/release/unity.md (hunk 3 above)
Comment (rajendraindukuri) [reply content/docs/csidriver/release/unity.md:11]: done

File: content/docs/csidriver/installation/operator/unity.md [hunk 4]
```diff
@@ -107,12 +107,12 @@ metadata:
   namespace: unity
 spec:
   driver:
-    configVersion: v2.7.0
+    configVersion: v2.8.0
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/unity.md:110]: If my understanding is correct, there would be no release for dell-csi-operator in CSM 1.8. So you could revert the changes in this file completly

**Review by shanmydell — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Review by rensyct — APPROVED** [review]

**Review by chimanjain — APPROVED** [review]

**Review by sharont58 — APPROVED** [review]
